src/main/java/com/revature/controllers/SwaggerfileController.java

```diff
@@ -3,96 +3,77 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.List;
-import java.util.Map;
+import java.util.concurrent.Executors;
+import javax.persistence.EntityManager;
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.revature.models.Docutest;
+import com.revature.models.Request;
+import com.revature.models.SwaggerDocutest;
+import com.revature.models.SwaggerSummary;
+import com.revature.models.SwaggerUploadResponse;
 import com.revature.services.JMeterService;
-import io.swagger.models.HttpMethod;
-import io.swagger.models.Model;
-import io.swagger.models.Operation;
+import com.revature.services.OASService;
+import com.revature.services.ResultSummaryService;
+import com.revature.services.SwaggerSummaryService;
+import com.revature.templates.LoadTestConfig;
 import io.swagger.models.Swagger;
-import io.swagger.models.parameters.BodyParameter;
-import io.swagger.models.parameters.HeaderParameter;
-import io.swagger.models.parameters.Parameter;
-import io.swagger.models.parameters.PathParameter;
-import io.swagger.models.properties.Property;
 import io.swagger.parser.SwaggerParser;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.multipart.MultipartFile;
 
-
-
 @CrossOrigin(origins = "*", allowedHeaders = "*")
 @Controller
```

Comment:
This method likely should be annotated with `@RestController` instead. Otherwise all of the methods below would need `@ResponseBody`